### PR TITLE
ACS-8132: send timestamp to hxi

### PR DESCRIPTION
**Pull request by Pawel-608**

### CODE REVIEW

File: live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/delete/DeleteNodeCommand.java [hunk 1]
```diff
@@ -27,10 +27,13 @@
...
 
-public record DeleteNodeCommand(String nodeId)
+public record DeleteNodeCommand(
+        String nodeId,
+        long timestamp)
 {
     public DeleteNodeCommand
     {
         ensureNotBlank(nodeId, "Node id cannot be blank.");
+        // ensureThat(timestamp > 0, "Timestamp cannot be negative or zero.");
```
Comment (krdabrowski) [inline live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/delete/DeleteNodeCommand.java:37]: Uncomment or remove this code

File: live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/metadata/IngestNodeCommand.java [hunk 2]
```diff
@@ -37,12 +37,14 @@
...
-        Set<PropertyDelta<?>> properties)
+        Set<PropertyDelta<?>> properties,
+        long timestamp)
 {
     public IngestNodeCommand
     {
         ensureNotBlank(nodeId, "Node id cannot be blank");
         ensureNonNull(eventType, "Node %s event type cannot be null", nodeId);
         ensureNonNull(properties, "Node %s properties delta cannot be null", nodeId);
+        // ensureThat(timestamp > 0, "Timestamp cannot be negative or zero.");
```
Comment (krdabrowski) [inline live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/metadata/IngestNodeCommand.java:48]: Here the same as above

**Review by krdabrowski — APPROVED** [review]
LGTM - I have only minor comments

File: live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/delete/DeleteNodeCommand.java (hunk 1 above)
Comment (Pawel-608) [reply live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/delete/DeleteNodeCommand.java:37]: nice catch! I commented it for test purposes

**Pull request merged**